charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml

```
@@ -41,6 +41,10 @@ spec:
             - --host={{ include "ingress-nginx.controller.fullname" . }}-admission,{{ include "ingress-nginx.controller.fullname" . }}-admission.{{ .Release.Namespace }}.svc
             - --namespace={{ .Release.Namespace }}
             - --secret-name={{ include "ingress-nginx.fullname" . }}-admission
+{{- with .Values.controller.admissionWebhooks.patch.resources }}
+          resources:
+{{ toYaml . | indent 12 }}
+{{- end }}
```

Please match our formatting throughout the chart. Also, make the same change on the other template as well

```suggestion
        {{- with .Values.controller.admissionWebhooks.patch.resources }}
          resources: {{ toYaml . | nindent 12 }}
        {{- end }}
```